packages/editor-ui/src/api/workflows.ts

```diff
@@ -28,7 +28,9 @@ export async function getActiveWorkflows(context: IRestApiContext) {
 }
 
 export async function getActiveExecutions(context: IRestApiContext, filter: IDataObject) {
-	return await makeRestApiRequest(context, 'GET', '/executions/active', { filter });
+	const output = await makeRestApiRequest(context, 'GET', '/executions', { filter });
```

Comment:
This will be removed right? This is just here to prevent breaking things entirely I guess

Reply:
The old endpoint does not exist anymore, so to test on the UI I had to update it, but Alex has the final say on this.